default.nix

```diff
@@ -128,7 +128,7 @@ in rec {
   obelisk = ghcObelisk;
   obeliskEnvs = ghcObeliskEnvs;
   command = ghcObelisk.obelisk-command;
-  shell = pinBuildInputs "obelisk-shell" ([command] ++ commandRuntimeDeps pkgs) [];
```

Comment:
Not sure what this `[]` was supposed to do, but it causes `error: attempt to call something which is not a function but a set`

Reply:
Probably some older nixpkgs thing that went away when bumped.